evalml/pipelines/utils.py

```diff
@@ -610,11 +611,12 @@ def make_pipeline(
     return pipeline
 
 
-def generate_pipeline_code(element):
+def generate_pipeline_code(element, features_path=None):
     """Creates and returns a string that contains the Python imports and code required for running the EvalML pipeline.
 
     Args:
         element (pipeline instance): The instance of the pipeline to generate string Python code.
+        features_path (str): path to features json. Defaults to None.
```

Comment:
Nit: can we clarify that this should be a json created from featuretools? It could be confusing to users as is.